From 71bcbf852e6d4434b56dba88f3a0146d80648e79 Mon Sep 17 00:00:00 2001
From: Vlad Zahorodnii <vlad.zahorodnii@kde.org>
Date: Fri, 14 Oct 2022 12:22:44 +0300
Subject: [PATCH] wayland: Sync output device's enabled property

The output can be already disabled when the output device is created.

BUG: 460247
---
 src/wayland/outputdevice_v2_interface.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/wayland/outputdevice_v2_interface.cpp b/src/wayland/outputdevice_v2_interface.cpp
index 788f6cba55..4ff9f971fa 100644
--- a/src/wayland/outputdevice_v2_interface.cpp
+++ b/src/wayland/outputdevice_v2_interface.cpp
@@ -164,6 +164,7 @@ OutputDeviceV2Interface::OutputDeviceV2Interface(Display *display, KWin::Output
     : QObject(parent)
     , d(new OutputDeviceV2InterfacePrivate(this, display, handle))
 {
+    updateEnabled();
     updateManufacturer();
     updateEdid();
     updateUuid();
-- 
GitLab

